client/src/components/CommentApp/components/CommentReply/style.scss

```diff
@@ -2,111 +2,13 @@
   padding-top: 20px;
   pointer-events: auto;
   position: relative;
-  border-top: 1px solid $color-comment-separator;
+  border-top: 1px solid theme('colors.border-furniture');
 
-  &__text {
-    color: $color-box-text;
-    font-weight: 400;
-    font-size: 16px;
-    line-height: 150%;
-    margin-bottom: 0;
-    padding-top: 10px;
-    padding-bottom: 10px;
-    word-break: break-all;
-    white-space: pre-wrap;
-
-    &--mode-deleting {
-      color: theme('colors.text-context');
-    }
-  }
-
-  &--mode-deleting &__avatar {
-    opacity: 0.5;
-  }
-
-  &--mode-deleting &__text {
+  &--mode-deleting .comment__text {
     color: theme('colors.text-meta');
   }
 
   form {
     margin-top: 10px;
   }
-
-  &__button {
-    @include button;
```

Comment:
Everything here and below unless flagged is 100% the same as the styles for the `Comment` component.